aria-practices.html

```diff
@@ -5807,6 +5807,7 @@ <h2>Grid and Table Properties</h2>
       <li>Whether any columns or rows are hidden, e.g., columns 1 through 3 and  5 through 8 are visible but column 4 is hidden.</li>
       <li>Whether a cell spans multiple rows or columns.</li>
       <li>Whether and how data is sorted.</li>
+      <li>How to communicate cell position, e.g., <q>row 2, column 2</q> or <q>row 2, column B</q> (like in most spread sheets).
```

Comment:
No space in "spreadsheets".

Reply:
But how is it a real spreadsheet if it isn’t spread by a space? 🤔